codepile/enwikibooks/README.md

```diff
@@ -0,0 +1,19 @@
+### Download WikiBook dumps dataset 
+
+```bash
+wget https://dumps.wikimedia.org/enwikibooks/20220901/enwikibooks-20220901-pages-meta-current.xml.bz2 -O data/enwikibooks-20220820-pages-meta-current.xml.bz2
```

Comment:
make sure to add `mkdir data` before this incase the person hasn't created the folder